src/NodaTime.Web/Views/Shared/_Layout.cshtml

```diff
@@ -50,28 +50,28 @@
                     <li class="has-dropdown">
                         <a href="#">User Guide</a>
                         <ul class="dropdown">
-                            <li><label>Versions</label></li>
+                            <li><label>Current</label></li>
+                            <li><a href="/1.3.x/userguide/">1.3.x</a></li>
+                            <li><a href="/2.0.x/userguide/">2.0.x</a></li>
+                            <li><label>Other</label></li>
```

Comment:
Maybe "Old"? And should this be under "Development", as the old releases are arguably less important?